tutorials/gSlack/index.md

```diff
@@ -0,0 +1,172 @@
+# gSlack
```

Comment:
I would also suggest a title that explains the contents of the tutorial better in a few more words. gSlack name may even step into a territory with trademark issues.